python/nano/src/bigdl/nano/deps/ipex/ipex_inference_model.py

```diff
@@ -150,7 +153,7 @@ def __init__(self, model: torch.nn.Module, input_sample=None, use_ipex=False, dt
                                                          strict=jit_strict)
                         except Exception:
                             self.model = torch.jit.script(self.model)
-                    self.model = torch.jit.freeze(self.model)
+                    # self.model = torch.jit.freeze(self.model)
```

Comment:
this line is commented, I understand it's called in `_load` if user load the model, but when it's called when user call `trace`

Reply:
will remove it, it's a mistake here.